### DISCUSSION TITLE
WG assignments for projects

### DISCUSSION THREAD

**Discussion by ReesePlews** [ISO-TC211/StandardsTracker#512] [General]
hello @PeterParslow and @JanHjelmager  

i added WG labels to a few of the issues that came in but in checking the project's WG assignments [here](https://committee.iso.org/sites/tc211/home/projects/projects-per-working-group.html) i noticed that some are slated to be moved, once a revision restart would begin. 

i created a new label "WG assignment pending" and added that label to one of the issues because it was clear the WG for the project was going to change but had not changed yet (based on the web page above). 

assignment of these labels is not a critical bit of operation for the repository but it is nice if we could add them. i think it is just basic type of house-keeping for the issues.

if you both agree or have other advice/opinions for doing this, please add to the discussion. looking forward to hearing from you both.

**Comment by PeterParslow** [comment]
Reese, that seems to be a good approach. Our (my) assumption is that a standard can only be moved from one WG to another when there is an active project, so it seems wise to flag them in the Standards Tracker.

**Comment by ReesePlews** [comment]
thank you Peter, so in the list of projects if a specific project is marked as "moved" then we can assign the WG, but if not then it would be the "pending" label.   i will do a few and for ones i am not sure if just leave them. i will see what Jan says or just get started on the easy ones... i dont think we are in a hurry.  thanks for the feedback.

**Reply by JanHjelmager** [reply]
I agree with the both of you. So thank you Reese and Peter

**Reply by ReesePlews** [reply]
thanks @JanHjelmager i will start to look at them and make the assignments as i have time.
